docs/Typechecking.md

```diff
@@ -40,11 +40,12 @@ out the resulting type errors. Read more about Sorbet's strictness levels
 ## Using `brew typecheck`
 
 When run without any arguments, `brew typecheck`, will run considering the strictness levels
-set in the `files.yaml` file. However, when typecheck is run on a specific file
-or directory, more errors may show up since Sorbet can not resolve constants defined
-outside the scope of the specified file. These problems can be solved with RBI files.
-Currently `brew typecheck` provides `quiet`, `--file`, `--dir` and `--ignore` options
-but you can explore more options with `srb tc --help` and passing them with `srb tc`.
+set in each of the individual Ruby files in the core Homebrew codebase. However, when
+typecheck is run on a specific file or directory, more errors may show up since Sorbet
+cannot resolve constants defined outside the scope of the specified file. These
+problems can be solved with RBI files. Currently `brew typecheck` provides `quiet`, `--file`,
```

Comment:
Shouldn't 'quiet' also be noted as a flag?

```suggestion
problems can be solved with RBI files. Currently `brew typecheck` provides `--quiet`, `--file`,
```

Reply:
Thanks!